src/components/appointmentsTable/renderAppointments.tsx

```diff
@@ -0,0 +1,19 @@
+import { TableRow, Alert } from '@mui/material';
+import { Appointment } from './types';
+import RowTable from './tableRow';
+import { message } from './style';
+
+const renderAppointments = (appointments: never[]) => (appointments?.length

```

Comment:
It's a more convenint in this wat to create a react component that return 
```jsx
 <RowTable
      key={appointment.id}
      appointment={appointment}
    />
```

and at the parent itself do map through data and handle different cases